Sources/Memcache/ChannelHandler/MemcacheDecodingError.swift

```diff
@@ -0,0 +1,52 @@
+import NIOCore
+
+struct MemcacheDecodingError: Error {
+    let messageVerb: String
+    let payload: String
+    let description: String
+    let file: StaticString
+    let line: UInt
+
+    static func withPartialError(
+        _ partialError: MemcachePartialDecodingError,
+        messageVerb: String,
+        messageBytes: ByteBuffer
+    ) -> Self {
+        MemcacheDecodingError(
+            messageVerb: messageVerb,
+            payload: "", // TODO: Can we get a base64 representation without Foundation?
```

Comment:
Maybe it makes sense to read the readable bytes as a string since the protocol is string-based. Might produce weird output because the data block of value responses can be arbitrary data.